src/config/onCreateNode.ts

```diff
@@ -0,0 +1,16 @@
+import { createFilePath } from 'gatsby-source-filesystem'
+import _ from 'lodash'
+
+export const onCreateNode = ({ node, actions, getNode }) => {
+  const { createNodeField } = actions
+  const type: string = _.get(node, 'internal.type')
+
+  if (type === 'Directory' || type === 'MarkdownRemark') {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)